### ISSUE TITLE
TypeError: Cannot set property finalTitle of null

### ISSUE THREAD

**Issue by ghost** [doZennn/steam-rom-manager#308]
After the update from yesterday (perhaps, I haven't used it in like 4 days), I haven't been able to get my parser config working. Any tips?

```New config preset file has been downloaded.
All available image urls retrieved.
Executing parsers.
Fatal error occurred in Preview. See event log for details.
Error: File Parser Execution:
 Apply user exceptions step for Sony PlayStation 3 - RPCS3 (Installed PKG):
 TypeError: Cannot set property 'finalTitle' of null```

**Comment by cbartondock** [comment]
Well hey at least the improved logging I added is working lol. Looking into it.

**Comment by cbartondock** [comment]
In the meantime I assume it works if you disable the PS3 parser - what games are there?

**Comment by cbartondock** [comment]
I think I found the issue - it would occur if you both exclude a title and try to give it a new title in userExceptions. Bit of an odd thing to do, but is that what you are doing?

**Comment by ghost** [comment]
I'll give it a shot! The titles were identical for everything I put in exceptions, so like 
Field 1: Game4
Field 2: Game4

**Comment by cbartondock** [comment]
Oh that's not how you use the exceptions manager! If the fields are blank it does nothing, so just leave them blank if you want to keep them the same.

**Comment by ghost** [comment]
Okay!

**Comment by ghost** [comment]
That works!!!

**Comment by cbartondock** [comment]
Okay! I am going to leave this open until I release 2.3.38 which contains the fix (assuming someone does want to set a title and do an exclude for some odd reason). Thanks for the catch!

**Comment by ghost** [comment]
Thank you so much!!

**Comment by cbartondock** [comment]
Fixed in 2.3.38.
